en/sidebars.json

```diff
@@ -165,11 +165,29 @@
             "monitoring/event-handler",
             "monitoring/web-import-export"
         ],
-        "Alerts & Notifications": [
-            "alerts-notifications/concepts",
-            "alerts-notifications/resources-status",
-            "alerts-notifications/event-console",
-            "alerts-notifications/manage-alerts",
+        "Managing events and alerts": [    
+            {
+                "ids":[
+                    "alerts-notifications/concepts",
+                    "alerts-notifications/resources-status",
+                    "alerts-notifications/event-console",
+                    "alerts-notifications/event-log",
+                    "alerts-notifications/custom-views"             
+                ],
+                "label": "Viewing events",
+                "type": "subcategory"
+            },  
+            {   
+                "ids" : [
+                    "alerts-notifications/acknowledge",
+                    "alerts-notifications/downtimes",
+                    "alerts-notifications/submit",
+                    "alerts-notifications/notif-flapping",
```

Comment:
This is both times, first here but no present into table of contant and secondly in "Managing alerts"